### ISSUE TITLE
ModuleNotFoundError: No module named 'audio_separator'

### ISSUE THREAD

**Issue by Echolink50** [AIFSH/ComfyUI-Hallo#7]
Getting this error ModuleNotFoundError: No module named 'audio_separator'
I have the audio_separator folder with kim_vocal_2.onnx file and some other files. Is audio_separator something that needs to bee installed with python or what? Thanks

**Comment by AIFSH** [comment]
try py310

**Comment by Echolink50** [comment]
> try py310

Thanks. I got past that part with "pip install audio-separator==0.12.3"
Then I ran into no insightface module so I downloaded insightface-0.7.3-cp311-cp311-win_amd64.whl and installed
Then I ran into no xformers. Thats where i'm currently stuck. My comfyui is the latest running python 3.11, torch 2.3.1, cu121. I downloaded xformers-0.0.27.dev792-cp311-cp311-win_amd64.whl and installed. It downgraded my torch to 2.3.0 and now comfyui doesn't start. So now I am trying to upgrade back to 2.3.1 and use xformers@git+https://github.com/facebookresearch/xformers.git@main#egg=xformers to hopefully install and run with any torch version.

Do I need to downgrade comfy to python 3.10? And what torch and cu?
